test/IDE/complete_modulename.swift

```diff
@@ -0,0 +1,53 @@
+// BEGIN MyModule.swift
+public struct MyModuleTy {}
+public func myModuleFunc() {}
+
+
+// BEGIN _Hidden.swift
+public struct HiddenTy {}
+public func hiddenFunc() {}
+
+// BEGIN App.swift
+
+import MyModule
+import _Hidden
+
+func test() {
+ let _ = #^EXPR^#
+
+ func test() -> #^TYPE^#
+}
+
+// EXPR: Begin completion
+// EXPR-NOT: _Concurrency[#Module#]
+// EXPR-NOT: SwiftShims[#Module#]
+// EXPR-NOT: SwiftOnoneSupport[#Module#]
+// EXPR-DAG: Decl[Module]/None:                  swift_ide_test[#Module#]; name=swift_ide_test
+// EXPR-DAG: Decl[Module]/None/IsSystem:         Swift[#Module#]; name=Swift
+// EXPR-DAG: Decl[Module]/None:                  MyModule[#Module#]; name=MyModule
+// EXPR-DAG: Decl[Struct]/OtherModule[MyModule]: MyModuleTy[#MyModuleTy#]; name=MyModuleTy
+// EXPR-DAG: Decl[Struct]/OtherModule[_Hidden]:  HiddenTy[#HiddenTy#]; name=HiddenTy
+// EXPR-DAG: Decl[FreeFunction]/OtherModule[MyModule]: myModuleFunc()[#Void#]; name=myModuleFunc()
+// EXPR-DAG: Decl[FreeFunction]/OtherModule[_Hidden]: hiddenFunc()[#Void#]; name=hiddenFunc()
+// EXPR: End completions
+
+// TYPE: Begin completion
+// TYPE-NOT: _Concurrency[#Module#]
+// TYPE-NOT: SwiftShims[#Module#]
+// TYPE-NOT: SwiftOnoneSupport[#Module#]
```

Comment:
What about _Hidden? I actually think we should show it here since it is explicitly imported.

Reply:
Oh, I think I just forgot to add a line for `_Hidden`.
Anyway, let's make it show explicitly imported modules regardless.

Reply:
Added a commit. How does it look?